frontend/src/scenes/billing/BillingProduct.tsx

```diff
@@ -513,7 +512,7 @@ export const BillingProduct = ({ product }: { product: BillingProductV2Type }):
                                                 </div>
                                             ) : null}
                                         </>
-                                    ) : (
+                                    ) : product.current_amount_usd ? (
                                         <div className="my-8">
                                             <Tooltip
                                                 title={`The current amount you will be billed for this ${billing?.billing_period?.interval}.`}
```

Comment:
```suggestion
                                                title={`The amount you will be billed for this ${billing?.billing_period?.interval}.`}
```

Reply:
I think 'current' is important in this context because it's not what they will be charged at the end of the period.